implementation/routing/src/routing_manager_stub.cpp

```diff
@@ -1084,6 +1084,13 @@ routing_manager_stub::on_net_state_change(
 
 void routing_manager_stub::on_offer_service(client_t _client,
         service_t _service, instance_t _instance, major_version_t _major, minor_version_t _minor) {
+
+    VSOMEIP_DEBUG << "ON_OFFER_SERVICE("
```

Comment:
Please use func name, for example:
VSOMEIP_DEBUG << __func__ << ": ..

Reply:
Done! https://github.com/COVESA/vsomeip/pull/728/commits/421131148d678aff6708d710239fd70ce948ed5e